File: sod/sod-schedule-center/sod-schedule-dao/src/main/java/com/piesat/schedule/mapper/test/TestMapper.java
package com.piesat.schedule.mapper.test;

import com.piesat.schedule.entity.test.Test;
import org.springframework.stereotype.Component;

import java.util.List;

/**
 * @program: sod
 * @description:
 * @author: zzj
 * @create: 2020-02-07 11:28
 **/
@Component
public interface TestMapper {
    List<Test> findList();
}

File: sod/sod-monitor-center/sod-monitor-server/src/main/scala/com/piesat/monitor/dao/es/TestMapper.scala
package com.piesat.monitor.dao.es

import org.apache.ibatis.annotations.Mapper
import java.util.{List => JavaList}

import com.piesat.monitor.entity.Test

/**
  * Created by zzj on 2020/3/19.
  */
@Mapper
trait TestMapper {

  def list():JavaList[Test]

}
